packages/react/src/field/Field.tsx

```diff
@@ -37,17 +43,32 @@ export const Field = forwardRef<HTMLDivElement, FieldProps>(
     return (
       <Flex flexDirection="column" gap="2" maxW="sm" ref={ref} {...props}>
         {label && (
-          <Text asChild>
-            <RadixLabel.Root htmlFor={id}>
-              {label}
-              {required && (
-                <Text aria-hidden="true" as="span" color="fg.error">
-                  {" "}
-                  *
-                </Text>
-              )}
-            </RadixLabel.Root>
-          </Text>
+          <Flex flexDirection="row" gap="4">
+            <Text
+              alignItems="center"
+              asChild
+              display="flex"
+              fontWeight="500"
+              gap="2"
+            >
+              <RadixLabel.Root htmlFor={id}>
+                {label}
+                {required && (
+                  <Text aria-hidden="true" as="span" color="fg.error">
+                    *
+                  </Text>
+                )}
+              </RadixLabel.Root>
+            </Text>
+            {info && (
+              <>
+                <Tooltip content={info}>
+                  <IconInfo aria-details="iconInfo"></IconInfo>
+                </Tooltip>
+                <VisuallyHidden.Root id="iconInfo">{info}</VisuallyHidden.Root>
```

Comment:
let's use dynamic ID here instead of static one - to handle multiple fields with info on same page
```suggestion
                  <IconInfo aria-details={infoId}></IconInfo>
                </Tooltip>
                <VisuallyHidden.Root id={infoId}>{info}</VisuallyHidden.Root>
```
```tsx
const infoId = useId();
```